terraform/README.md

```diff
@@ -124,24 +163,31 @@ If you are setting up a new AWS account to be managed by terraform (eg. a Simple
 follow these instructions. This setup needs to be run only once per AWS account.
 
 - Create an AWS account.
-- The repo contains a separate directory for each AWS account. For setting this up with a new account,
```

Comment:
[maybe] Move this section to the top? (before `Getting Started`)

Reply:
This info (one dir per AWS org) is already in the overview. I think it should suffice.